src/Parser/Warrior/Arms/Modules/Core/Execute.js

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+
+import { formatPercentage } from 'common/format';
+import SpellLink from 'common/SpellLink';
+import SPELLS from 'common/SPELLS';
+import Wrapper from 'common/Wrapper';
+
+import Analyzer from 'Parser/Core/Analyzer';
+import Enemies from 'Parser/Core/Modules/Enemies';
+
+import SpellUsable from '../Features/SpellUsable';
+
+class ExecutionAnalyzer extends Analyzer {
+  static dependencies = {
+    enemies: Enemies,
+    spellUsable: SpellUsable,
+  };
+
+  executes = 0;
+  wastedExecutionersPrecisions = 0;
+
+  on_byPlayer_cast(event) {
+    if(SPELLS.EXECUTE.id !== event.ability.guid) {
+      return;
+    }
+
+    this.executes += 1;
+    const enemy = this.enemies.getEntity(event);
+    const executionersPrecision = enemy.getBuff(SPELLS.EXECUTIONERS_PRECISION.id, null, 0, 0, event.sourceID);
```

Comment:
No need to include other parameters than the spellID, we don't track other players debuffs on enemies since we only parse events that include the player.